app/src/main/java/com/winlator/contents/ContentProfile.java

```diff
@@ -16,6 +16,7 @@ public class ContentProfile {
     public static final String MARK_WINE_BINPATH = "binPath";
     public static final String MARK_WINE_LIBPATH = "libPath";
     public static final String MARK_WINE_PREFIX_PACK = "prefixPack";
+    public static final String MARK_PROTON = "proton";
```

Comment:
Checked as a fallback when parsing the profile.json for extraction ( some profile.json have their /binPath and /libPath keys underneath a "proton" parent key.)